.github/issue_template.md

```diff
@@ -0,0 +1,9 @@
+Please be sure to include:
+
+* what operating system you are developing on (linux or macOS)
+
+* for macOS, what version of Xcode you are using (`xcodebuild -version`)
+
+* what branch/tag of SwiftProtobuf you are using (1.0.0, etc.)
+
+* what version of Swift is your code set to compile with
```

Comment:
Maybe just "the version of Swift you are using" with a callout to `swift --version`

In any case, I would put this before the SwiftProtobuf version.  There's a natural order here:  either OS -> dev tools -> project   or   project -> dev tools -> OS.

Reply:
The catch is the version of swift they use does not have to the same version that the library is compiled with (or that `swift --version`) will report.  Xcode settings, Package.swift info, etc all seems to come into play.

About the only thing I thought I was sure about was `swift --version` would only tell us if they had a custom version installed, but even if they do, that doesn't mean Xcode ends up using it.

Anyway, generally tweaked.